### PR TITLE
Личный проект: пока все дома

### PR DESCRIPTION
**Pull request by akzhar**
---
:mortar_board: [Личный проект: пока все дома](https://up.htmlacademy.ru/javascript/17/user/490939/tasks/5)

:boom: https://htmlacademy-javascript.github.io/490939-keksobooking-17/2/

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+var NUMBER_OF_OFFERS = 8;
+var PIN_WIDTH = 50;
+var PIN_HEIGHT = 70;
+var mapBlock = document.querySelector('.map');
+var mapPins = mapBlock.querySelector('.map__pins');
+var templatePin = document.getElementById('pin').content.querySelector('.map__pin');
+var apartmentTypes = ['palace', 'flat', 'house', 'bungalo'];
+
+function getRandomElemInArr(arr) {
```
Comment (rkhaslarov) [inline js/main.js]: Take a look to the previous task =)

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  return arr[Math.round(Math.random() * (arr.length - 1))];
+}
+
+function getRandomNumber(min, max) {
+  return Math.round(Math.random() * (max - min) + min);
+}
+
+function createsimmilarAppartments(n) {
```
Comment (rkhaslarov) [inline js/main.js]: camelCase? And they're similar? )

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        x: getRandomNumber(0, 1200),
+        y: getRandomNumber(130, 630)
+      }
+    };
+  }
+  return simmilarAppartments;
+}
+
+function renderPin(appartment) {
+  var pin = templatePin.cloneNode(true);
```
Comment (rkhaslarov) [inline js/main.js]: appartment -> apartment

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+var simmilarAppartments = createsimmilarAppartments(NUMBER_OF_OFFERS);
+var fragment = document.createDocumentFragment();
+for (var i = 0; i < simmilarAppartments.length; i++) {
+  var pin = renderPin(simmilarAppartments[i]);
+  fragment.appendChild(pin);
+}
+mapPins.appendChild(fragment);
+mapBlock.classList.remove('map--faded');
```
Comment (rkhaslarov) [inline js/main.js]: isolate the code into the main function =)

File: js/main.js (hunk 2 above)
Comment (akzhar) [reply js/main.js]: fixed

File: js/main.js (hunk 3 above)
Comment (akzhar) [reply js/main.js]: fixed

File: js/main.js (hunk 1 above)
Comment (akzhar) [reply js/main.js]: done

File: js/main.js (hunk 4 above)
Comment (akzhar) [reply js/main.js]: done)

**Pull request merged**